templates/single.twig

```diff
@@ -14,7 +14,7 @@
 			<div class="container">
 				<div class="top-page-tags">
 					{% if ( page_type ) %}
-						<a class="tag-item tag-item--main page-type" href="{{ fn('get_home_url') ~ '/page_type/' ~ page_type.slug }}">{{ page_type|e('wp_kses_post')|raw }}</a>
+						<a class="tag-item tag-item--main page-type" href="{{ fn('rtrim', fn('get_home_url'), '/') ~ '/page_type/' ~ page_type.slug }}">{{ page_type|e('wp_kses_post')|raw }}</a>
```

Comment:
We could probably encapsulate this into a helper but maybe it's out of the scope of this issue.

Reply:
Actually I saw that with the rewrite rule in place for p4-page-type taxonomy we can use built-in WordPress functions.

[get_term_link](https://developer.wordpress.org/reference/functions/get_term_link/)
will return the full url for the term (and it will work with wpml enabled also)

https://www.planet4.test/en/page_type/story/
https://www.planet4.test/page_type/story/

so we can use `fn('get_term_linkl', page_term_id)` passing the term_id/object/slug as parameter.

Reply:
@kirdia Good point, updated! Also in the blocks plugin: https://github.com/greenpeace/planet4-plugin-blocks/pull/375